### DISCUSSION TITLE
customize create button in filament

### DISCUSSION THREAD

**Discussion by HShoja786** [filamentphp/filament#8248] [Help]
customize create button in filament

**Comment by leandrocfe** [comment]
You can customize in the ListPage
```php
//ListPage.php
Actions\CreateAction::make()
    ->label('new label')
    ->icon('heroicon-s-plus')
```
If you want to override the create button in the CreatePage, you can do this

```php
//CreatePage.php
protected function getCreateFormAction(): \Filament\Actions\Action
{
    return parent::getCreateFormAction()
        ->label('new label')
        ->icon('heroicon-s-plus');
}
```

**Reply by Cepairda** [reply]
> You can customize in the ListPage
> 
> ```
> //ListPage.php
> Actions\CreateAction::make()
>     ->label('new label')
>     ->icon('heroicon-s-plus')
> ```
> 
> If you want to override the create button in the CreatePage, you can do this
> 
> ```
> //CreatePage.php
> protected function getCreateFormAction(): \Filament\Actions\Action
> {
>     return parent::getCreateFormAction()
>         ->label('new label')
>         ->icon('heroicon-s-plus');
> }
> ```

How i can use mutate ? This is not working

```php
 parent::getCreateFormAction()
                ->label('Draft')
                ->mutateFormDataUsing(function (array $data): array {
                    $data['status'] = OrderStatus::Draft;
                    return $data;
                }),
```
